packages/elements/src/select/index.ts

```diff
@@ -295,6 +321,28 @@ export class Select extends ControlElement implements MultiValue {
     return this.selectedSlotItems.map(item => this.getItemValue(item));
   }
 
+  /**
+   * Aria indicating that select has a popup of type listbox
+   * @ignore
+   */
+  @property({ type: String, reflect: true, attribute: 'aria-haspopup' })
+  public ariaHasPopup = 'listbox';
```

Comment:
Overriding AOM model does not work in Firefox (the properties are not reflected). It has to work directly with attributes (via setAttribute/removeAttribute). Probably we need a new util for that.

Reply:
On my testing, Firefox 94.0.2 does reflect these attributes correctly just like any other lit properties. Can you recheck again ?